test/TypeScript/README.md

```diff
@@ -1,2 +1,19 @@
 # Library of TypeScript tests
 To run the entire test suite, execute `./gradlew test --tests org.lflang.tests.runtime.TypeScriptTest.*`.
+
+## Using an alternative runtime
+To run the tests with an alternative runtime, use the `-Druntime` flag to specify where to find it.
+
+### Examples
+- To use a local checkout of `reactor-ts` located in the local file system in the directory `~/lf-lang/reactor-ts`:
+```
+./gradlew test --tests org.lflang.tests.runtime.TypeScriptTest.* -Druntime="~/lf-lang/reactor-ts"
+```
+- To point to a particular ref (e.g., `feature-branch`) of the `reactor-ts` repository:
+```
+./gradlew test --tests org.lflang.tests.runtime.TypeScriptTest.* -Druntime="git://github.com/lf-lang/reactor-ts.git#feature-branch"
+```
+- Note that `lfc` can be pointed to an alternative runtime as well, using the `external-runtime-path` switch:
+```
+lfc test/TypeScript/src/Minimal.lf --external-runtime-path ~/lf-lang/reactor-ts
```

Comment:
Note that for lfc we also have the `--runtime-version` argument which is intended for specifying a particular ref to be fetched from the main runtime repository. Is this also supported for TS now? If not, I think we should add it.

Reply:
OK, I've added this (and removed the overloading of the `-Druntime` switch as it 1) isn't strictly necessary and 2) doesn't seem to allow `pnpm` to reuse prior installations of `@lf-lang/reactor-ts`. Instead, in CI, I'm just checking out the repo and pointing to the location in the filesystem for tests invoked as part of `lf-lang/reactor-ts` CI runs.